agent/consul/leader_peering_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/base64"
 	"encoding/json"
+	"github.com/hashicorp/consul/sdk/freeport"
```

Comment:
nit: import ordering

Reply:
Sounds good. Should we add a linter for this?